react-native-shimmer.podspec

```diff
@@ -15,5 +17,6 @@ Pod::Spec.new do |s|
   s.requires_arc    = true
   s.platform        = :ios, "9.0"
 
-  s.dependency 'React'
+  s.dependency 'React', '>= 0.45.1'
```

Comment:
This one is a nice way to get rid of the React 0.12 dependency, wonder why I haven't thought about this before. :-)